sklearn/feature_selection/_univariate_selection.py

```diff
@@ -236,8 +236,13 @@ def r_regression(X, y, *, center=True, force_finite=True):
     This is a scoring function to be used in a feature selection procedure, not
     a free standing feature selection procedure.
 
-    The cross correlation between each regressor and the target is computed
-    as ((X[:, i] - mean(X[:, i])) * (y - mean_y)) / (std(X[:, i]) * std(y)).
+    This is done in 2 steps:
+
+    1. The correlation between each regressor and the target is computed,
+       that is, mean((X[:, i] - mean(X[:, i])) * (y - mean(y))) / (std(X[:, i])
+       * std(y)).
+
+    2. It is converted to an F score then to a p-value.
```

Comment:
Currently this diff is in `r_regression`. I think the diff should be in `f_regression`?

(There is no p-value computed in `r_regression`)

Reply:
Huh, that's weird. I fixed it now. Not sure how that happened.